pkg/workers/exec/konnector_test.go

```diff
@@ -176,8 +176,96 @@ echo "{\"type\": \"manifest\", \"message\": \"$(ls ${1}/manifest.konnector)\" }"
 		wg.Done()
 	}()
 
+	wg.Wait()
+	wg.Add(1)
+	msg, err := jobs.NewMessage(map[string]interface{}{
+		"konnector": "my-konnector-1",
+	})
+	assert.NoError(t, err)
+
+	j := jobs.NewJob(inst, &jobs.JobRequest{
+		Message:    msg,
+		WorkerType: "konnector",
+	})
+
+	config.GetConfig().Konnectors.Cmd = tmpScript
+	ctx := jobs.NewWorkerContext("id", j).WithCookie(&konnectorWorker{})
+	err = worker(ctx)
+	assert.NoError(t, err)
+
+	wg.Wait()
+}
+
+func TestSecretFromAccountType(t *testing.T) {
+	script := `#!/bin/bash
+
+SECRET=$(echo "$COZY_PARAMETERS" | sed -e 's/.*secret"://' -e 's/[},].*//')
```

Comment:
What does the sed do ?

Reply:
It extracts the secret (as a string) from the JSON of the COZY_PARAMETER env variable